.github/workflows/pr-quality-gate.yaml

```diff
@@ -45,13 +45,11 @@ jobs:
 
       - name: Split providers by concurrency needs
         id: split-providers
-        # TODO: stop splitting providers based on a hard coded value. Maybe
-        # in the future something like `vunnel list --only-concurrent` could
-        # supply the values here. Hard code for now.
         run: |
-          all_providers='${{ steps.determine-providers.outputs.providers }}'
-          multicore_providers=$(echo "$all_providers" | jq -c '[.[] | select(. == "ubuntu")]')
-          other_providers=$(echo "$all_providers" | jq -c '[.[] | select(. != "ubuntu")]')
+          cd tests/quality
+          # use vunnel's tag system to split providers by concurrency needs
+          multicore_providers=$(make select-providers TAG=multicore)
```

Comment:
How does this relate to `determine-providers` (the preceding step?). In other words `all_providers='${{ steps.determine-providers.outputs.providers }}'` disappears so I'm confused.